<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>600</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QComboBox" name="resumeTypeBox">
                <property name="geometry">
                    <rect>
                        <x>80</x>
                        <y>40</y>
                        <width>191</width>
                        <height>25</height>
                    </rect>
                </property>
            </widget>
            <widget class="QCheckBox" name="scenesDetectionBox">
                <property name="geometry">
                    <rect>
                        <x>90</x>
                        <y>160</y>
                        <width>141</width>
                        <height>22</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Scenes detection</string>
                </property>
            </widget>
            <widget class="QSlider" name="scenesDifSlider">
                <property name="geometry">
                    <rect>
                        <x>290</x>
                        <y>160</y>
                        <width>160</width>
                        <height>17</height>
                    </rect>
                </property>
                <property name="minimum">
                    <number>1</number>
                </property>
                <property name="maximum">
                    <number>100</number>
                </property>
                <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                </property>
                <property name="invertedAppearance">
                    <bool>false</bool>
                </property>
                <property name="invertedControls">
                    <bool>false</bool>
                </property>
            </widget>
            <widget class="QPushButton" name="nextButton">
                <property name="geometry">
                    <rect>
                        <x>650</x>
                        <y>460</y>
                        <width>87</width>
                        <height>29</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Next</string>
                </property>
            </widget>
            <widget class="QPushButton" name="previousButton">
                <property name="geometry">
                    <rect>
                        <x>80</x>
                        <y>470</y>
                        <width>87</width>
                        <height>29</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Previous</string>
                </property>
            </widget>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>800</width>
                    <height>23</height>
                </rect>
            </property>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <resources/>
    <connections/>
</ui>
